packages/mobile/src/screens/profile-screen/ProfileHeader/TopSupporters.tsx

```diff
@@ -97,6 +98,14 @@ export const TopSupporters = () => {
     })
   }, [navigation, user_id])
 
+  useLayoutEffect(() => {
```

Comment:
Let's add a comment here explaining this is for controlling a loading animation!